### PR TITLE
feat: improve helm installation process

### PR DESCRIPTION
**Pull request by Piotr1215**
## Summary

Improve clarity and usability of the `helm` installation documentation instructions.

Docs Link: https://deploy-preview-285--vcluster-docs-site.netlify.app/docs/platform/install/helm

Closes DOC-268, DOC-274

### CODE REVIEW

File: platform/install/helm.mdx
Comment (deniseschannon) [inline platform/install/helm.mdx:166]: Does this make sense at the bottom of the page or should it be higher up before you install? Based on our defaults, it feels like I would deploy and then have to update the config and re-deploy. 

If we keep it at the bottom, I would talk about the config but also link to a section (which might not exist yet) on how to redeploy. 

Something similar to this page that we have in vcluster docs: https://vcluster.com/docs/vcluster/manage/deploy-changes

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: platform/install/helm.mdx
Comment (Piotr1215) [reply platform/install/helm.mdx:127]: That's a good idea :+1:

File: platform/install/helm.mdx
Comment (Piotr1215) [reply platform/install/helm.mdx]: I mentioned the `loft-router`. Do you think more detailed information is needed about the router? Maybe we could add a dedicated doc in the `Understand` section?

File: platform/install/helm.mdx
Comment (Piotr1215) [reply platform/install/helm.mdx:131]: I created is as partial, but the part with secret retrieval is `helm` specific. Using CLI for example wouldn't require secret retrieval. This method assumes that `vcluster CLI` is not available.

**Review by deniseschannon — COMMENTED** [review]
@Piotr1215 I've read through this and there are little things that would improve the overall clarity of this document. 

1. Update any reference of "vCluster Platform" to use "the platform". 
2. ~We should only refer to the configuration file as the `platform configuration file`. I see references using `vcluster.yaml` and `values.yaml` to indicate the same file.~ I totally misunderstood the differences, but cleaned it up as there was some confusion in here. 

There are other areas that I might nitpick on, but I think it's more of a user preference than anything else.

**Comment by deniseschannon** [comment]
Updated Preview: https://deploy-preview-285--vcluster-docs-site.netlify.app/docs/platform/install/helm/

**Comment by deniseschannon** [comment]
@sowmyav27 As a new user, could you also read through the most recent deploy preview to see if this makes sense?

File: platform/install/helm.mdx
Comment (Piotr1215) [inline platform/install/helm.mdx:20]: I'm curious why we are recommending minor version as well and not just `v3`?

File: platform/install/helm.mdx
Comment (Piotr1215) [inline platform/install/helm.mdx:82]:
```suggestion
Copy the default values file and modify it to suite their deployment needs, or create
```

File: platform/install/helm.mdx
Comment (deniseschannon) [reply platform/install/helm.mdx:20]: Honestly, I'm not sure. It's a requirement that was listed on vcluster docs, but never asked. 

@FabianKramm Do you have any ideas on why the minor version would matter? 

@Piotr1215 I don't have a preference either way, but would like the pre-reqs between vcluster and vcluster platform docs to match.

**Review by deniseschannon — COMMENTED** [review]
Can we add bash titles throughout this doc like you did in the Install vCluster CLI section? I really like them for the people who are going to skim and just read /copy the commands.

File: platform/install/helm.mdx
Comment (FabianKramm) [reply platform/install/helm.mdx:20]: I think we used capabilities for a while which was I believe a feature they added later on, but since we removed that I guess its fine to just say v3+

File: platform/install/helm.mdx
Comment (Piotr1215) [reply platform/install/helm.mdx]: This is moved now, and the configuration section is in the middle of the flow.

**Review by deniseschannon — APPROVED** [review]
Overall, this is looking great. The only addition would be adding how to create vclusters (like in here: https://deploy-preview-291--vcluster-docs-site.netlify.app/docs/platform/install/quick-start-guide/#create-virtual-clusters)

I don't want to hold up the approval for that last change.

**Pull request merged**
